ls1intum/artemis-android
# `Improvement`: Re-design Lecture Overview

### Problem Description
The previous lecture UI displayed all content on a single page, resulting in long scrolls and a cluttered user experience. This layout was not aligned with the iOS app's design, where lecture details are displayed in a bottom sheet for better organization and user interaction.

### Changes
 

- Redesigned the lecture UI to display details in a bottom sheet instead of a single scrolling page.
- Made lecture unit headers clickable to trigger the bottom sheet.
- Bottom sheet dynamically shows content based on the selected unit (e.g., text, video, etc).
- Added scroll support for content overflow and improved typography, spacing, and alignment.

### Steps for testing

- Navigate to the lecture overview screen in the Android app.
- Verify that the headers for each lecture unit are displayed correctly.
- Click on a header to open the lecture details in a bottom sheet.
- Ensure the bottom sheet content is rendered appropriately for different lecture unit types (e.g., text, video, etc).

### Screenshots
Old and new version added for easier comparison

https://github.com/user-attachments/assets/42150565-7626-4585-bac3-bc27c207d5fb
https://github.com/user-attachments/assets/a889b0fb-66fb-41de-b989-d318da818408
